ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Translation.hs

```diff
@@ -31,17 +31,17 @@ data EraTranslation xs = EraTranslation {
   deriving NoThunks
        via OnlyCheckWhnfNamed "EraTranslation" (EraTranslation xs)
 
-translateTxIn ::
+hTranslateTxIn ::
      All Top xs
   => EraTranslation xs
   -> InPairs TranslateTxIn xs
-translateTxIn = InPairs.hmap (TranslateTxIn . translateTxInWith) . translateLedgerTables
+hTranslateTxIn = InPairs.hmap (TranslateTxIn . translateTxInWith) . translateLedgerTables
 
-translateTxOut ::
+hTranslateTxOut ::
      All Top xs
   => EraTranslation xs
   -> InPairs TranslateTxOut xs
-translateTxOut = InPairs.hmap (TranslateTxOut . translateTxOutWith) . translateLedgerTables
+hTranslateTxOut = InPairs.hmap (TranslateTxOut . translateTxOutWith) . translateLedgerTables
```

Comment:
Maybe `ipTranslateTxIn` and `ipTranslateTxOut`?

Reply:
Done